crowbar_framework/app/models/api/node.rb

```diff
@@ -48,6 +48,14 @@ def os_upgrade
       false
     end
 
+    def router_migration
+      if execute_and_wait_for_finish("/usr/sbin/crowbar-router-migration.sh", 600)
```

Comment:
If we use this method (`execute_and_wait_for_finish`), the script needs to create those `-ok` and `-failed` files. Also, the timeout might be shorted, I believe.